src/common/tse.c

```diff
@@ -1010,6 +1010,14 @@ tse_task_reinit(tse_task_t *task)
 	dtp->dtp_running = 0;
 	dtp->dtp_completing = 0;
 	dtp->dtp_completed = 0;
+
+	/** reset stack pointer as zero */
+	if (dtp->dtp_stack_top != 0) {
+		D_ERROR("task %p, dtp_stack_top reset from %d to zero.\n",
```

Comment:
what would cause this?

Reply:
like push to stack but forgot to pop, here just print a err and reset it.